axelor-dailyts/src/main/resources/views/TimesheetLine.xml

```diff
@@ -5,12 +5,12 @@
 
 	<grid name="timesheet-line-daily-timesheet-grid" title="Daily timesheet lines"
 		model="com.axelor.apps.hr.db.TimesheetLine" editable="true"
-		onNew="action-timesheet-line-attrs-set-default-onnew" orderBy="-teamTask.id">
+		onNew="action-timesheet-line-attrs-set-default-onnew" orderBy="-projectTask.id">
 
 		<field name="project" required="true" form-view="project-form"
 			grid-view="project-grid" onChange="action-timesheet-line-group-project-onchange"
 			onSelect="action-timesheet-line-attrs-domain-project" />
-		<field name="teamTask" form-view="team-task-form" grid-view="team-task-grid"
+		<field name="projectTask" form-view="team-task-form" grid-view="team-task-grid"
 			onSelect="action-timesheet-line-attrs-domain-team-task"
```

Comment:
rename action-timesheet-line-attrs-domain-team-task to action-timesheet-line-attrs-domain-project-task